src/Flag.js

```diff
@@ -7,28 +7,36 @@ import Box from './Box'
 
 const capitalize = str => str.charAt(0).toUpperCase() + str.slice(1)
 
-const darkBorderColor = props => {
+const shadowColor = props => {
   const darkColor = theme(`colors.dark${capitalize(props.color)}`)(props)
+
+  if (!darkColor) {
```

Comment:
props to @jxnblk for your suggestion. This is much better than adding a darkBgColor prop 🙇‍♀️